### DISCUSSION TITLE
Not able to config the integration

### DISCUSSION THREAD

**Discussion by christophgasser** [astrandb/miele#179] [General]
Hi,

I had the "old" Miele integration installed and it suddenly stopped working. I took the chance to upgrade to the new version but can't get it configured. That's what I did:

1. Removed the "Miele" part from the configuration.yaml
2. Removed the old integration via HACS
3. Installed the new integration via HACS
4. Restarted HA

When trying to add the integration in HA I just get the message "Die Komponente ist nicht konfiguriert. Bitte folge der Anleitung.". Which means "Component not configured. Please follow the documentation". I really tried hard to find anything in the documentation, but could not fin anything.

Can someone please give me a hint, what I am doing wrong?

BR
Christoph

**Comment by astrandb** [comment]
You still need the `miele:` entry in configuration.yaml with client_id and client_secret. The same setup as with the old integration.

I am sorry that the description on the front page is a little ahead of time. My mistake. A new feature will come in next release where you enter these credentials in the UI.
You can download the pre-release through HACS, select re-download of this integration and select version `main` at the bottom of the list-box. Restart HA and Add the integration Miele.

**Reply by christophgasser** [reply]
Uups. Could have thought about that myself. Sorry. Working now.
But thank you for the quick response! :-)
